astropy/modeling/core.py

```diff
@@ -3067,12 +2950,10 @@ def inputs_map(self):
                         inputs_map[inp] = self.left, self.left.inputs[i]
                 else:  # Get from right
                     if isinstance(self.right, CompoundModel):
-
-                        inputs_map[inp] = r_inputs_map[self.right.inputs[i
-                                                                   - len(self.left.inputs)]]
```

Comment:
A bit weird in that two different changes made to line lengths that are only different by one. Seems like one or the other should be picked for consistency (visually anyway)